src/Microsoft.AspNetCore.Mvc.Core/Internal/ProblemDescriptionFilter.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Core;
+using Microsoft.AspNetCore.Mvc.ErrorDescription;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
+{
+    public class ProblemDescriptionFilter : IResultFilter
```

Comment:
This should be an action filter. Result filters aren't called in some cases

Reply:
Yup, missed updating it.